tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ClientCredentialsMtlsPopTests.cs

```diff
@@ -67,5 +67,53 @@ public async Task Sni_Gets_Pop_Token_Successfully_TestAsync()
             // Assert: Verify that the token was fetched from cache on the second request
             Assert.AreEqual(TokenSource.Cache, authResult.AuthenticationResultMetadata.TokenSource, "Token should be retrieved from cache");
         }
+
+        [DoNotRunOnLinux] // POP is not supported on Linux
+        [TestMethod]
+        public async Task BindingCertificate_Is_Surfaced_In_SniFlow_And_Null_In_BearerFlow_TestAsync()
```

Comment:
Integration tests are expensive. Do not add new tests for minor features. Please update existing integration tests with new asserts.